app/__init__.py

```diff
@@ -7,3 +7,4 @@
 INVALID_CODE_MSG = {'text': 'Please re-enter your access code and try again.', "clickable": True, "level": "ERROR", "type": "INVALID_CODE"}  # NOQA
 NOT_AUTHORIZED_MSG = {'text': 'There was a problem connecting to this study. Please try again later.', "level": "ERROR", "type": "SYSTEM_AUTH_ERROR"}  # NOQA
 ADDRESS_CHECK_MSG = {'text': 'Please check and confirm address.', "level": "ERROR", "type": "ADDRESS_CONFIRMATION_ERROR"}
+ADDRESS_EDIT_MSG = {'text': 'Enter address to continue', "level": "ERROR", "type": "ADDRESS_EDIT_ERROR"}
```

Comment:
Is it possible to have these messages ready for translation e.g. to prepare for internationalisation?

Reply:
There is a requirement for display of pages in welsh but as far as I am aware no requirement has been put forward for any other language at present. I am looking at how to do this in Python for RH but it would be delivered as a separate feature branch.